x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/index.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { CreateEngine } from './create_engine';
```

Comment:
Very opinionated ask - can we rename the **folder** to `engine_creation/` vs `create_engine/`? I'm 100% fine with the file/component name remaining as-is, but from a finder/directory perspective it's way easier to locate this when it's alphabetically listed right next to `engine/`.

FWIW we also do this with `documents/` and `document_creation/`

Reply:
I will do this, but lololololol this has actually tripped me up like 5000 times in the past two weeks because I navigate everything using the `cmd+P` file finder in VSCode and I kept typing out `createdocument` and it wouldn't return any files based on how it handles its autocompletion. But I'll go with the flow!

Reply:
`document_creation` was `DocumentCreation` and not `CreateDocument` even in the standalone UI so I'm a little confused on your meaning there - the file finder in `ent-search` doesn't return anything for me for `createdocument` either 🤔 

The file name remaining `create_engine.ts` should still be cmd+P-able at least!